homeassistant/components/webostv/__init__.py

```diff
@@ -48,6 +49,9 @@
                         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
                         vol.Optional(CONF_ON_ACTION): cv.SCRIPT_SCHEMA,
                         vol.Optional(CONF_ICON): cv.string,
+                        vol.Optional(
+                            CONF_CONSECUTIVE_VOLUME_STEPS_DELAY
```

Comment:
I don't think we need an option for this. Just set a sane default.

Reply:
> Is this behavior specific for volume changes only?

I'm not sure actually. Could perhaps be the same for channel up/down, but I haven't been able to test that.

> I don't think we need an option for this. Just set a sane default.

Alright, removed the config option